lib/matplotlib/__init__.py

```diff
@@ -948,7 +948,7 @@ def rc_params_from_file(fname, fail_on_error=False, use_default_template=True):
 
     Parameters
     ----------
-    fname : str
+    fname : str, path-like
```

Comment:
```suggestion
    fname : str or path-like
```

Reply:
According to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections), which we follow, all types are separated by `or` not commas (also multiple ones `A or B or C`).